.gitignore

```diff
@@ -1 +1,4 @@
-node_modules/
\ No newline at end of file
+node_modules/
+
+tests/logs/
+tests/pacts/
```

Comment:
not putting these in version control, though i could be swayed that the `pacts/` dir should be included.